### ISSUE TITLE
Extract PS2 games with version names

### ISSUE THREAD

**Issue by Cancelling66** [israpps/HDL-Batch-installer#95]
### Explanation

I changed my hard drive and wanted to extract my games from the old HDD to transfer them to the new one, but your software does not extract the version name, like USA or Europe, so it just leaves the simple name of the game.

### fature related

- [ ] game Installation
- [X] Title Database
- [ ] ART Downloader
- [X] Game management
- [X] HDD management
- [ ] Others...

### other checks

- [ ] I've checked if such feature exists on [Latest version](https://github.com/israpps/HDL-Batch-installer/releases/)
- [ ] This Feature requires an update of WxWidgets
- [ ] Needs an update of [HDL Dump](https://github.com/ps2homebrew/hdl-dump)

**Comment by israpps** [comment]
you can Identify the region by looking at the game ID wich will be automatically detected by the program as soon as the game is installed again.

**Comment by Cancelling66** [comment]
Yes I know, but when you extract an ISO from the program and there are two games with the same name, but different region it poses a problem

**Comment by Cancelling66** [comment]
Add the extraction of the original ISO date, which is 12/24/1996 at 11:32 p.m.
